### PR TITLE
Added mailing notification to site admins when outage is created

### PR DESCRIPTION
**Pull request by sparavalo**

### CODE REVIEW

File: settings.php [hunk 1]
```diff
@@ -29,6 +29,10 @@
  */
 use auth_outage\local\outagelib;
 
+// print_object(get_admins());
```
Comment (gnat) [inline settings.php]: If you're not using it, get rid of it.

File: settings.php [hunk 2]
```diff
@@ -29,6 +29,10 @@
...
+
+// get_admins();
```
Comment (gnat) [inline settings.php]: If you're not using it, get rid of it.

**Comment by gnat** [comment]
@sparavalo Looks good aside from the left in debug lines.

**Comment by sparavalo** [comment]
@gnat 
Can you please check new changes?
Thanks

**Comment by gnat** [comment]
Looks good! @sparavalo

**Pull request merged**